sagemaker-experiments/sagemaker_job_tracking/pytorch_distributed_training_experiment.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.             hyperparameters={
</pre><p>Nit: remove extra spaces. </p><p>Running the linter may help to reformat each files: https://github.com/aws/amazon-sagemaker-examples/blob/main/CONTRIBUTING.md#run-the-linter</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3703/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>